src/OrganisationRegistry.ElasticSearch/Organisations/OrganisationDocument.OrganisationRegulation.cs

```diff
@@ -81,6 +84,10 @@ public static IPromise<IProperties> Mapping(PropertiesDescriptor<OrganisationReg
                         t => t
                             .Name(p => p.Url)
                             .Fields(x => x.Keyword(y => y.Name("keyword"))))
+                    .Text(
+                        t => t
+                            .Name(p => p.WorkRulesUrl)
+                            .Fields(x => x.Keyword(y => y.Name("keyword"))))
```

Comment:
this means only full matches on the entire URI will be matched. I'm not sure what the desired behavior is, might need to check with analysts. (non-blocking imo)